src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -0,0 +1,1024 @@
+      decomp_cascade_con%decomp_pool_name_restart(i_pas_som) = 'soil3'
+      decomp_cascade_con%decomp_pool_name_history(i_pas_som) = 'PAS_SOM'
+      decomp_cascade_con%decomp_pool_name_long(i_pas_som) = 'passive soil organic matter'
+      decomp_cascade_con%decomp_pool_name_short(i_pas_som) = 'S3'
+      is_microbe(i_pas_som) = .false.
+      is_litter(i_pas_som) = .false.
+      is_soil(i_pas_som) = .true.
+      is_cwd(i_pas_som) = .false.
+      initial_cn_ratio(i_pas_som) = 10._r8  ! cn_s3 in BGC; not used in MIMICS
+      initial_stock(i_pas_som) = params_inst%initial_Cstocks(i_pas_som)
+      is_metabolic(i_pas_som) = .false.
+      is_cellulose(i_pas_som) = .false.
+      is_lignin(i_pas_som) = .false.
+
+      i_cop_mic = i_pas_som + 1
+      floating_cn_ratio_decomp_pools(i_cop_mic) = .true.
+      decomp_cascade_con%decomp_pool_name_restart(i_cop_mic) = 'micr1'
+      decomp_cascade_con%decomp_pool_name_history(i_cop_mic) = 'COP_MIC'
+      decomp_cascade_con%decomp_pool_name_long(i_cop_mic) = 'copiotrophic microbes'
+      decomp_cascade_con%decomp_pool_name_short(i_cop_mic) = 'M1'
+      is_microbe(i_cop_mic) = .true.
+      is_litter(i_cop_mic) = .false.
+      is_soil(i_cop_mic) = .false.
+      is_cwd(i_cop_mic) = .false.
+      initial_cn_ratio(i_cop_mic) = 10._r8  ! MIMICS may use this
+      initial_stock(i_cop_mic) = params_inst%initial_Cstocks(i_cop_mic)
+      is_metabolic(i_cop_mic) = .false.
+      is_cellulose(i_cop_mic) = .false.
+      is_lignin(i_cop_mic) = .false.
+
+      i_oli_mic = i_cop_mic + 1
+      floating_cn_ratio_decomp_pools(i_oli_mic) = .true.
+      decomp_cascade_con%decomp_pool_name_restart(i_oli_mic) = 'micr2'
+      decomp_cascade_con%decomp_pool_name_history(i_oli_mic) = 'OLI_MIC'
+      decomp_cascade_con%decomp_pool_name_long(i_oli_mic) = 'oligotrophic microbes'
+      decomp_cascade_con%decomp_pool_name_short(i_oli_mic) = 'M2'
+      is_microbe(i_oli_mic) = .true.
+      is_litter(i_oli_mic) = .false.
+      is_soil(i_oli_mic) = .false.
+      is_cwd(i_oli_mic) = .false.
+      initial_cn_ratio(i_oli_mic) = 10._r8  ! MIMICS may use this
+      initial_stock(i_oli_mic) = params_inst%initial_Cstocks(i_oli_mic)
+      is_metabolic(i_oli_mic) = .false.
+      is_cellulose(i_oli_mic) = .false.
+      is_lignin(i_oli_mic) = .false.
+
+      if (.not. use_fates) then
+         ! CWD
+         i_cwd = i_oli_mic + 1
+         floating_cn_ratio_decomp_pools(i_cwd) = .true.
+         decomp_cascade_con%decomp_pool_name_restart(i_cwd) = 'cwd'
+         decomp_cascade_con%decomp_pool_name_history(i_cwd) = 'CWD'
+         decomp_cascade_con%decomp_pool_name_long(i_cwd) = 'coarse woody debris'
+         decomp_cascade_con%decomp_pool_name_short(i_cwd) = 'CWD'
+         is_microbe(i_cwd) = .false.
+         is_litter(i_cwd) = .false.
+         is_soil(i_cwd) = .false.
+         is_cwd(i_cwd) = .true.
+         initial_cn_ratio(i_cwd) = 10._r8  ! 90 in BGC; not used in MIMICS
```

Comment:
all initial_cn_ratio terms need to be 1
all rf_decomp_cascade need to be 0
for the decomp_k loop to work in soilbiogeochem/SoilBiogeochemPotentialMod.F90
but does it break anything else and is this how we want to resolve this...?

Reply:
I looked again at the decomp_k loop in `SoilBiogeochemPotentialMod.F90`, and I think it's fine as is.